seqr/utils/elasticsearch/es_gene_agg_search.py

```diff
@@ -12,9 +14,9 @@ class EsGeneAggSearch(EsSearch):
     CACHED_COUNTS_KEY = None
 
     def aggregate_by_gene(self):
-        searches = {self._search}
+        searches = [self._search]
         for index_searches in self._index_searches.values():
-            searches.update(index_searches)
+            searches = searches + [index_search for index_search in index_searches if index_search not in searches]
```

Comment:
I can't make `{self._search}` work for Python 3 because I can't modify the `Search` class of `Elasticsearch`. Using a `list` can be a workaround for this case.

Reply:
I know, but given that sets don't work means that identity checking doesn't work, which means the line you added here that says `if index_search not in searches` shouldn't work either, so why is it here?

Reply:
The base class of `Search` which is `Request` has an `__eq__` method. The method is used when identity is being checked.